### PR TITLE
LPS-158851 [RAYLIFE AP] Getting data from coverage

### PR DESCRIPTION
**Pull request by EugenioAntunes**

### CODE REVIEW

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by EugenioAntunes** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @liferay/imports-first */
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:1]: Delete this line, please.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx [hunk 2]
```diff
@@ -130,7 +163,7 @@ const Review = () => {
 
 						<ClayTable.Cell className="col" headingCell>
 							<span className="text-dark">
-								<h6>$0.00</h6>
+								<h6>$611.00</h6>
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx:166]: I believe that you should not mock this value. It should be a sum of the values of the column.

**Review by joanagrego — CHANGES_REQUESTED** [review]
I added some suggestions :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 3]
```diff
@@ -14,7 +15,12 @@
 
 import {ClaySelect} from '@clayui/form';
 import classNames from 'classnames';
+import {useContext} from 'react';
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:19]: Please, remove this line.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 4]
```diff
@@ -166,7 +194,13 @@ const CoverageInfoForm = () => {
 
 			<div className="row">
 				<div className="col-sm filled form-condensed form-group">
-					<ClaySelect aria-label="Select Label" id="Comprehensive">
+					<ClaySelect
+						aria-label="Select Label"
+						id="Comprehensive"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:199]: Please, change it to lowercase 
```
id="comprehensive"
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 5]
```diff
@@ -186,7 +220,13 @@ const CoverageInfoForm = () => {
 				</div>
 
 				<div className="col-sm filled form-condensed form-group">
-					<ClaySelect aria-label="Select Label" id="Collision">
+					<ClaySelect
+						aria-label="Select Label"
+						id="Collision"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:225]: Please, change the id value to lowercase

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx [hunk 6]
```diff
@@ -13,47 +13,81 @@
...
 const Review = () => {
+	const [state] = useContext(NewApplicationAutoContext);
+
 	const coverages = [
 		{
 			name: 'BODILY INJURY',
-			premium: '$0.00 ',
-			protectionAmount: '$0.00 / $0.00',
+			premium: '$167.00 ',
+			protectionAmount: `${state.steps.coverage.form.bodilyInjury}`,
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx:27]: Please, change it to 
```
form.*
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx [hunk 7]
```diff
@@ -13,47 +13,81 @@
...
-		{
-			name: 'VEHICLE 1: COLLISION',
-			premium: '$0.00 ',
-			protectionAmount: '$0.00 / $0.00',
+			premium: '$115.00 ',
+			protectionAmount: `${state.steps.coverage.form.medical}`,
 		},
 	];
 
+	const vehiclesTable = state.steps.vehicleInfo.form.map((v, index) => (
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx:51]: Please, change it to ```vehicleInfo.form.map((vehicleInfoForm, index) =>```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx [hunk 8]
```diff
@@ -13,47 +13,81 @@
...
+			<ClayTable.Row className="no-gutters row text-uppercase">
+				<ClayTable.Cell className="col-6" headingTitle>
+					<p className="text-secondary">
+						Vehicle {index + 1}: Comprehensive
+					</p>
+				</ClayTable.Cell>
+
+				<ClayTable.Cell className="col-4">
+					<p className="text-secondary">
+						{state.steps.coverage.form.comprehensive}
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx:62]: Please, change it to 
```
form.*
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions 🔥

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
LGTM :1st_place_medal: Thanks @EugenioAntunes

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Review by joanagrego — APPROVED** [review]
Nice job! :up:

**Comment by EugenioAntunes** [comment]
ci:test:relevant

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**